pkg/hs/urbit-king/lib/Urbit/Vere/Ports.hs

```diff
@@ -69,6 +69,14 @@ portLeaseLifetime = 15 * 60
 portRenewalTime :: Word32
 portRenewalTime = portLeaseLifetime - 60
 
+-- Number of retries before we give up on performing nat operations.
+maxRetries :: Int
+maxRetries = 3
+
+-- How long to wait between retries.
+networkRetryDelay :: Int
+networkRetryDelay = 5 * 1000000
```

Comment:
if you like you can put _s in the number to make its size more clear? Looks like it's 10e6?